modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,327 @@
+if (!$user->hasPermission('electrophysiology_browser_view_allsites')
+    && !((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs')))
+    && $user->hasPermission('electrophysiology_browser_view_site'))
+) {
+    header('HTTP/1.1 403 Forbidden');
+    exit;
+}
+
+$response = getSessionData($_REQUEST['sessionID']);
+
+echo json_encode($response);
+
+/**
+ * Get the session data information.
+ *
+ * @param string $sessionID ID of the electrophysiology session
+ *
+ * @return array with the session information
+ */
+function getSessionData(string $sessionID)
+{
+    $db = \NDB_Factory::singleton()->database();
+
+    $query = 'SELECT 
+                DISTINCT(pf.SessionID) 
+              FROM physiological_file pf 
+                LEFT JOIN session s ON (s.ID=pf.SessionID) 
+                LEFT JOIN candidate c USING (CandID)
+                LEFT JOIN psc ON (s.CenterID=psc.CenterID) 
+                LEFT JOIN physiological_output_type pot 
+                  USING (PhysiologicalOutputTypeID) 
+              WHERE 
+                s.Active = "Y" 
+                AND pf.FileType IN ("bdf", "cnt", "edf", "set", "vhdr", "vsm") 
+              ORDER BY pf.SessionID';
+
+    $sessions            = $db->pselect($query, array());
+    $sessions            = array_column($sessions, 'SessionID');
+    $response['patient'] = getSubjectData($sessionID);
+    $response['database']    = array_values(getFilesData($sessionID));
+    $response['sessions']    = $sessions;
+    $currentIndex            = array_search($sessionID, $sessions);
+    $response['nextSession'] = isset($sessions[$currentIndex+1]) ?? '';
+    $response['prevSession'] = isset($sessions[$currentIndex-1]) ?? '';
+
+    return $response;
+}
+
+/**
+ * Get the subject data information.
+ *
+ * @param int $sessionID ID of the electrophysiology session
+ *
+ * @return array with the subject information
+ */
+function getSubjectData($sessionID)
+{
+    $subjectData = array();
+    $timePoint   = \NDB_Factory::singleton()->timepoint(intval($sessionID));
+    $candidate   = \NDB_Factory::singleton()->candidate($timePoint->getCandID());
+
+    $subjectData['pscid']       = $candidate->getPSCID();
+    $subjectData['dccid']       = $timePoint->getCandID();
+    $subjectData['visit_label'] = $timePoint->getVisitLabel();
+    $subjectData['sessionID']   = $sessionID;
+    $subjectData['site']        = $timePoint->getPSC();
+    $subjectData['dob']         = $candidate->getCandidateDoB();
+    $subjectData['sex']         = $candidate->getCandidateSex();
+    $subjectData['subproject']  = $timePoint->getData('SubprojectTitle');
+    $subjectData['output_type'] = $_REQUEST['outputType'];
+
+    return $subjectData;
+}
+
+/**
+ * Get the list of electrophysiology recordings with their recording information.
+ *
+ * @param int $sessionID ID of the electrophysiology session
+ *
+ * @return array with the file collection
+ */
+function getFilesData($sessionID)
```

Comment:
```suggestion
function getFilesData($sessionID): array
```

Also add int or string to sessionID, depending on previous comment.

Reply:
done